src/templates/base_register.html

```diff
@@ -79,15 +78,14 @@
     })
 });
 </script>
-
-{% endblock %}
-{% block extrajs %}
```

Comment:
Sorry for replying late. 
Actually this block is for specifying additional JS functionality in template files. Hope that you know how these block works.